README.linux.md

```diff
@@ -148,3 +138,12 @@ then run command `ldconfig`.
 
     EOF'
     ```
+4. Make sure that you add `/usr/local/lib` to `/etc/ld.so.conf`, then run:
+    ```bash
+    sudo ldconfig
+    ```
```

Comment:
/usr/local/lib is the part of libc installation and contained in the /etc/ld.so.conf.d/libc.conf which installed by libc-bin. Does it need to be made explicitly?

Reply:
You are right, I've checked it on a clean install of Ubuntu 22.04. It only requires running sudo ldconfig. We don't need to explicitly add /usr/local/lib to /etc/ld.so.conf. So I will remove this step from the common platform setup instructions.